client/post-editor/media-modal/edit/edit-canvas.jsx

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
```
./node_modules/.bin/eslint client/post-editor/media-modal/edit/edit-canvas.jsx 

/Users/andrew/Desktop/wp-calypso/client/post-editor/media-modal/edit/edit-canvas.jsx
   10:5   warning  "noop" is defined but never used               no-unused-vars
   22:27  warning  Unexpected space before function parentheses   space-before-function-paren
   27:4   warning  Missing semicolon                              semi
   30:27  warning  Unexpected space before function parentheses   space-before-function-paren
   37:37  warning  Unexpected space before function parentheses   space-before-function-paren
   43:29  warning  Unexpected space before function parentheses   space-before-function-paren
   51:21  warning  Unexpected space before function parentheses   space-before-function-paren
   58:26  warning  Unexpected space before function parentheses   space-before-function-paren
   66:30  warning  Unexpected space before function parentheses   space-before-function-paren
   70:18  warning  Unexpected space before function parentheses   space-before-function-paren
   73:28  warning  Strings must use singlequote                   quotes
   76:21  warning  Unexpected space before function parentheses   space-before-function-paren
   84:2   warning  Mixed spaces and tabs                          no-mixed-spaces-and-tabs
   84:51  warning  There must be a space inside this paren        space-in-parens
   85:18  warning  There must be a space inside this paren        space-in-parens
   88:6   warning  There must be a space inside this paren        space-in-parens
   94:21  warning  There must be a space inside this paren        space-in-parens
   94:22  warning  A space is required after ','                  comma-spacing
   94:24  warning  A space is required after ','                  comma-spacing
   94:37  warning  A space is required after ','                  comma-spacing
   94:52  warning  There must be a space inside this paren        space-in-parens
   96:22  warning  Variables cannot be used as translate strings  wpcalypso/i18n-no-variables
   96:34  warning  Infix operators must be spaced                 space-infix-ops
   96:51  warning  Infix operators must be spaced                 space-infix-ops
   98:46  warning  Infix operators must be spaced                 space-infix-ops
   99:51  warning  Infix operators must be spaced                 space-infix-ops
   99:73  warning  Infix operators must be spaced                 space-infix-ops
  103:18  warning  Unexpected space before function parentheses   space-before-function-paren

✖ 28 problems (0 errors, 28 warnings)
```

Recommend installing an ESLint plugin for your editor to more easily catch these types of issues: https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#eslint